src/libs/actions/Transaction.ts

```diff
@@ -31,8 +31,8 @@ function createInitialWaypoints(transactionID: string) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`, {
         comment: {
             waypoints: {
-                waypoint0: {},
-                waypoint1: {},
+                waypoint0: null,
+                waypoint1: null,
```

Comment:
This change has caused a [regression](https://github.com/Expensify/App/issues/29748) explained here 
 -> https://github.com/Expensify/App/issues/29748#issuecomment-1766511326